src/app/models/usermodel.ts

```diff
@@ -0,0 +1,140 @@
+import {useState, useEffect} from 'react';
```

Comment:
The [commit](https://github.com/openstax/os-webview/pull/2701/commits/39a7e7b398584a56b89291049dcf91da222ad17b#diff-b110bad287dbbf7205d8a5fcaddae805630860a2ed81990509d344a5d236bbaa) between extracting accounts-model and running prettier is useful for seeing what actually changed.